BaseTools/Plugin/WindowsCapsuleSupportHelper/WindowsCapsuleSupportHelper.py

```diff
@@ -0,0 +1,62 @@
+##
+# UefiBuild Plugin that supports Window Capsule files based on the
+# Windows Firmware Update Platform spec.
+# Creates INF, Cat, and then signs it
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Copyright (c) Microsoft Corporation.

For all files.

Reply:
In a new commit or is it okay to just amend the old commit to follow the current copyright?